src/cpp/src/icontinuous_batching.hpp

```diff
@@ -49,6 +49,7 @@ class ContinuousBatchingPipeline::IContinuousBatchingPipeline {
 
     bool m_is_chat_conversation = false;
     ChatHistory m_history;
+    std::vector<ov::Tensor> m_history_images;
```

Comment:
we need to clean this vector in `finish_chat`

Reply:
Done.